docroot/modules/custom/foia_users/foia_users.module

```diff
@@ -122,4 +178,25 @@ function foia_users_form_alter(&$form, $form_state, $form_id) {
   if ($form_id == ('user_form' || 'user_register_form')) {
     $form['#validate'][] = '_form_validation_foia_agency';
   }
+
+  if ($form_id === 'user_register_form') {
+    $current_user = User::load(\Drupal::currentUser()->id());
+    $current_user_roles = $current_user->getRoles();
+    // Check the Agency Manager checkbox.
+    if (in_array('agency_administrator', $current_user_roles)) {
+      $form['account']['roles']['#default_value'][] = 'agency_manager';
```

Comment:
Discussion point and not MVP: I'm wondering if we should remove this from the agency admin's own `/user/uid/edit` page. A bit confusing I think.
![image](https://user-images.githubusercontent.com/7820280/31035776-459c5904-a537-11e7-9364-3e279d3d0790.png)